lib/spack/spack/cmd/find.py

```diff
@@ -140,6 +140,12 @@ def setup_parser(subparser):
     subparser.add_argument(
         "--only-deprecated", action="store_true", help="show only deprecated packages"
     )
+    subparser.add_argument(
+        "--install-tree",
+        action="store",
+        default="all",
+        help="Install trees to query. 'all' (default), 'local', 'upstream', or upstream name",
```

Comment:
```suggestion
        help="Install trees to query: 'all' (default), 'local', 'upstream', upstream name or path",
```

Also, it says "install trees" but can it be comma separated? Would be nice.  Or is the "trees" just because "upstream" happens to mean potentially multiple upstreams?